support-frontend/test/utils/CheckoutValidationRulesTest.scala

```diff
@@ -317,6 +576,140 @@ class GuardianWeeklyValidationTest extends AnyFlatSpec with Matchers {
 
 object TestData {
 
+  val paymentSwitchesOnState: Switches = Switches(
+    OneOffPaymentMethodSwitches(On, On, On, On, On),
+    RecurringPaymentMethodSwitches(On, On, On, On, On, On, On, On, On),
+    SubscriptionsPaymentMethodSwitches(On, On, On),
+    SubscriptionsSwitches(On, On, On),
+    FeatureSwitches(On, On),
+    CampaignSwitches(On, On),
+    RecaptchaSwitches(On, On),
+  )
```

Comment:
See if you can reduce the amount of code in the test data class by adding and using this function
```suggestion
  def buildSwitches(
      recurringPaymentMethodSwitches: RecurringPaymentMethodSwitches =
        RecurringPaymentMethodSwitches(On, On, On, On, On, On, On, On, On),
      subscriptionsPaymentMethodSwitches: SubscriptionsPaymentMethodSwitches =
        SubscriptionsPaymentMethodSwitches(On, Off, On),
  ) = Switches(
    OneOffPaymentMethodSwitches(On, On, On, On, On),
    recurringPaymentMethodSwitches,
    subscriptionsPaymentMethodSwitches,
    SubscriptionsSwitches(On, On, On),
    FeatureSwitches(On, On),
    CampaignSwitches(On, On),
    RecaptchaSwitches(On, On),
  )
```